comment/api/urls.py

```diff
@@ -1,11 +1,14 @@
 from django.urls import path
 from rest_framework.urlpatterns import format_suffix_patterns
+
 from comment.api import views
+from comment.views import react
 
 urlpatterns = [
     path('comments/', views.CommentList.as_view(), name='comments-list'),
     path('comments/create/', views.CommentCreate.as_view(), name='comments-create'),
     path('comments/<int:pk>/', views.CommentDetail.as_view(), name='comment-detail'),
+    path('comments/react/<int:comment_id>/<str:reaction>', react, name='react')
```

Comment:
For more conventions could you please modify the url as follows:
```python
path('comments/<int:comment_id>/react/<str:reaction>/', react, name='react')
```

Reply:
okay. In the other `url` file that renders template views, we use the `app_name` as `comment` and here as `comments`. Is this expected? Also, there this convention has not been followed. Should I make the change there too or leave it for backward compatibility?

Reply:
Also we need to change the `react` url in the other file to look like my suggestion here. `edit` and `delete` are actions performed on the comment model directly while `react` comes from another model, therefore, I would use comment id right after `comments/` in case of react or any other action not related to the comment model directly.